src/modules/account/components/EmailActionField/EmailActionField.tsx

```diff
@@ -20,5 +22,29 @@ export default EmailActionField;
 const EmailActionFieldAltContent: VFC = () => {
   const { user } = useAuth();
 
-  return <>{user?.email}</>;
+  return (
+    <>
+      <Text tw="mb-sm">{user?.email}</Text>
+      {!user?.emailVerified && <EmailUnverifiedAlert />}
+    </>
+  );
+};
+
+const EmailUnverifiedAlert: VFC = () => {
+  const { t } = useTranslation();
+  return (
+    // TODO(vito): Use a warning alert here when the component is built
+    <div>
+      <Text>
+        <span tw="font-bold">{t('common:fields.alert.warning')}</span>:{' '}
```

Comment:
Should be a variant of Text.

Reply:
I'll make it varient `body1` but this should be the default anyway

Reply:
I meant the span inside should be "sub1" which has preset font-weight and text-body1